client/view/BrowserStats/browserStats.js

```diff
@@ -86,7 +87,7 @@ export function updateBrowsersStats(data) {
     iconCell.classList.add('BrowsersStat__td')
     let iconElem = document.createElement('img')
     iconElem.classList.add('BrowsersStat__icon')
-    iconElem.src = `/assets/browser_logos/${id}.svg`
+    iconElem.src = browsersIcons[id]
```

Comment:
We need to take care of the case when we do not have icon for some browser.

Can I Use sometimes adds browsers.

Reply:
Added the same TODO for wikipedia